tools/run_tests/performance/scenario_config.py

```diff
@@ -245,13 +255,16 @@ def scenarios(self):
             secure=secure,
             categories=[SCALABLE])
 
-        for channels in [1, 3, 10, 31, 100, 316, 1000]:
-          yield _ping_pong_scenario(
-              'cpp_protobuf_%s_unary_qps_unconstrained_%s_%d_channels' % (synchronicity, secstr, channels),
-              rpc_type='UNARY',
-              client_type='ASYNC_CLIENT', server_type='ASYNC_SERVER',
-              unconstrained_client=synchronicity, secure=secure,
-              categories=[SWEEP], channels=channels)
+        for channels in geometric_progression(1, 500, math.sqrt(10)):
+          for outstanding in geometric_progression(1, 20000, math.sqrt(10)):
+              if synchronicity == 'sync' and outstanding > 400: continue
```

Comment:
But now that we restart workers every round, let's leave it in and see how we go.